src/Elmah.Io.NLog/ElmahIoTarget.cs

```diff
@@ -63,11 +63,7 @@ protected override void Write(LogEventInfo logEvent)
 
         private string MachineName(LogEventInfo logEvent)
         {
-#if NET45
             return new MachineNameLayoutRenderer().Render(logEvent);
```

Comment:
yes, reason why I switched to net1.6 don't think it's required to support older versions of netstandard as you support net45 as well